gax-grpc/src/test/java/com/google/api/gax/grpc/GrpcCallContextTest.java

```diff
@@ -46,21 +46,17 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import org.junit.Rule;
 import org.junit.Test;
-import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 import org.mockito.Mockito;
 import org.threeten.bp.Duration;
 
 @RunWith(JUnit4.class)
 public class GrpcCallContextTest {
-  @Rule public ExpectedException thrown = ExpectedException.none();
 
-  @Test
+  @Test(expected = IllegalArgumentException.class)
```

Comment:
You can't simply remove these. They need to be replaced with a try-catch block and a fail.

Reply:
Thanks for pointing it out, PTAL